src/daemon/common.c

```diff
@@ -1091,6 +1091,11 @@ int parse_identifier_vl (const char *str, value_list_t *vl) /* {{{ */
 
 int parse_value (const char *value_orig, value_t *ret_value, int ds_type)
 {
+  return parse_value_ext(value_orig, ret_value, ds_type, "");
+}
+
+int parse_value_ext (const char *value_orig, value_t *ret_value, int ds_type, const char *error_identifier)
```

Comment:
Not suite sure about `error_identifier`: it introduces a space in front of the colon in the common `parse_value()` case and is orthogonal to the `fgets()` change. To address the first complaint, you could turn this into `func_name` and, in `parse_value_file()`, do something along the lines of:

```c
char func_name[PATH_MAX];
ssnprintf (func_name, sizeof (func_name), "parse_value_file(\"%s\")", path);
/* … */
parse_value_ext (func_name, …);
```